### ISSUE TITLE
[CoE Starter Kit - QUESTION] Environments not showing in apps

### ISSUE THREAD

**Issue by GABRlEL** [microsoft/coe-starter-kit#8766]
### Does this question already exist in our backlog?

- [X] I have checked and confirm this is a new question.

### What is your question?

Hello there, first of all apologies if this question has already been asked - but I genuinely can't find it. In the Power Platform Admin View & DLP Impact Analysis apps, I can't choose any of my environments. Did I setup something wrong?

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

4.33

### What app or flow are you having the issue with?

Power Platform Admin View

### What method are you using to get inventory and telemetry?

Cloud flows

**Comment by Jenefer-Monroe** [comment]
Hello, 

Do you not even see environments you own yourself? 
Did you run the setup wizard after you installed Core?

**Comment by GABRlEL** [comment]
> Hello, 
> 
> Do you not even see environments you own yourself? 
> Did you run the setup wizard after you installed Core? 
> 

Hey there, thanks for getting in touch!
I'm by no means an expert in Power Platform but I'm system admin in the environment with CoE and I created it. The account I'm using is a Global Admin too.
I can see all of our 3 envis in the Platform admin center.

And yes I did run the wizard and finish it.

Best regards, Gabriel

**Comment by Jenefer-Monroe** [comment]
OK I see now. And don't worry we are here to help. 

Likely you have been impacted by new product behavior that shipped recently around the way the product treats privileged roles (ex Power Platform Admin role, Global Admin role)

While there is a workaround we can put into the kit to fix this directly, we cannot ship it with the kit until the workaround is available in all regions. Hopefully for the August release but we are still blocked by the roll out of this connector to some regions.
### The product change
Here is information about the product feature: [Manage admin roles with Microsoft Entra Privileged Identity Management](https://learn.microsoft.com/power-platform/admin/manage-high-privileged-admin-roles)

### How to check if this is the case
1. Validate the user running the flow has direct and permanent assignment to the Power Platform Admin role.
2. Take one of the target environments in your repro, one of the environments which is failing, and make sure the user running the flow has System Admin security role in that target environment.

### How to address and More information
Please see https://github.com/microsoft/coe-starter-kit/issues/8119 for a write up on this change. 
Included also is a workaround you can do until we can have it natively in the kit.

**Comment by GABRlEL** [comment]
> ### How to check if this is the case
> 1. Validate the user running the flow has direct and permanent assignment to the Power Platform Admin role.
> 2. Take one of the target environments in your repro, one of the environments which is failing, and make sure the user running the flow has System Admin security role in that target environment.
> 
> ### How to address and More information
> Please see #8119 for a write up on this change. Included also is a workaround you can do until we can have it natively in the kit.

Thanks for your input. I have verified that I'm a Global Admin & Power Platform Admin and I have the System Admin SR in the target environment. I have installed the workaround solution and activated the flow without errors.
Nothing changed for me - hence I get the slight suspicion I might be the issue. Any ideas?



> Also, since you are new to us, I thought I would share with you about our Office Hours
> 
> Please consider attending our monthly office hours. We talk about new features, upcoming changes, and spend most of the time answering your questions. More information: Join the [CoE Starter Kit Office hours](https://aka.ms/coeofficehours) held on the second Wednesday of each month between 7:00AM and 8:00AM Pacific time. Click here to download the calendar file to add to your calendar: [Calendar file](https://aka.ms/coeofficehours/ics)

Thanks for your offer, I'm not as invested into Power Platform yet but I'll def. keep it in mind!

**Comment by Jenefer-Monroe** [comment]
Please go to the most recent run of **_Admin | Sync Template v4 (Driver)_**

Are all the environments returned here?
![image](https://github.com/user-attachments/assets/13e0ecb4-d07f-4e8f-a125-443e2dc308bb)

**Comment by GABRlEL** [comment]
> Please go to the most recent run of **_Admin | Sync Template v4 (Driver)_**
> 
> Are all the environments returned here?
> ![image](https://github.com/user-attachments/assets/13e0ecb4-d07f-4e8f-a125-443e2dc308bb)
> 

Hey there, I think I understand the issue now. There's not a single run of this flow.
When running the setup wizard I had the problem that the sliders for enabling flows went back to Off after clicking them, no matter what I did. I thought that only was a visual issue but it probably wasn't. Could you please advise me on how to get all required flows up and running?

Best regards, Gabriel

**Comment by Jenefer-Monroe** [comment]
If the flow off? If so, please try to turn it on manually in the product itself, and see what error message you get.

**Comment by GABRlEL** [comment]
Yes, the flow is off, I assume that all necessary automates flows are. Is it sufficient to only enable the one flow? I'd like to make sure all necessary flows from the setup wizard are on. As I said, it seems like the setup was faulty for me

**Comment by Jenefer-Monroe** [comment]
No its not sufficient, but if it fails it will give you the error message so that you can see why turning these on is failing.

**Comment by Jenefer-Monroe** [comment]
closing out as no further action for starter kit team
